### PR TITLE
Adds alien tooltip style and updates operative tooltip style

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan:
tweak: Ghost revival/body creation alerts now use the ghost's hud style.
/:cl:

Alien!
Alien tooltip style matches alien speechbubbles.
Awwwwwperative!
Operative tooltip style matches the syndicate uplink.

### CODE REVIEW

**Comment by Supermichael777** [comment]
can we not have unreadable low contrast text? try a acid green and darker purple for the background

**Comment by KazeEspada** [comment]
The text looks fine to me and my eyes are starting to go bad.

**Comment by ChangelingRain** [comment]
Yeah that's not low-contrast, this is;
low, low contrast

**Comment by Supermichael777** [comment]
LIGHT TEXT AND DARK BACKGROUND OR DARK TEXT WITH LIGHT BACKGROUND.

**Comment by ChangelingRain** [comment]
That is a light background!!!

**Comment by Supermichael777** [comment]
maybe my eyes are just tired

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -168,6 +168,8 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		if(source)
 			var/obj/screen/alert/A = throw_alert("\ref[source]_notify_cloning", /obj/screen/alert/notify_cloning)
 			if(A)
+				if(client && client.prefs && client.prefs.UI_style)
+					A.icon = ui_style2icon(client.prefs.UI_style)
 				A.desc = message
```
Comment (KorPhaeron) [inline code/modules/mob/dead/observer/observer.dm]: Disgusting copypaste

**Pull request merged**